pycqed/tests/instrument_drivers/physical_instruments/golden/Test_QutechCC_test_all.scpi.txt

```diff
@@ -0,0 +1,53 @@
+*RST
```

Comment:
This test file does not belong in this location but rather in the test folder where we (intend to) mimic the folder structure of the main repo.

Reply:
it *is* in the 'tests' folder, this is the first instrument driver with a test